src/bson.rs

```diff
@@ -1054,6 +1054,99 @@ impl Binary {
     }
 }
 
+#[cfg(feature = "uuid-0_8")]
+#[cfg_attr(docsrs, doc(cfg(feature = "uuid-0_8")))]
+#[derive(PartialEq)]
+pub enum UuidRepresentation {
```

Comment:
we generally strive to add docstrings for all public types, so that includes this enum itself, each variant, and all its methods. You can use the spec and documentation from other drivers (e.g. [Java](https://mongodb.github.io/mongo-java-driver/4.3/apidocs/bson/org/bson/UuidRepresentation.html)) as inspiration for what the docstrings might contain.